joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/ControlStructureTests.scala

```diff
@@ -254,4 +252,37 @@ class ControlStructureTests extends RubyCode2CpgFixture {
     putsHi.lineNumber shouldBe Some(2)
   }
 
+  "`begin ... rescue ... end is represented by a `TRY` CONTROL_STRUCTURE node" in {
+    val cpg = code("""
```

Comment:
I think add multiple statements in at least one of these blocks to ensure they're correctly grouped for multiple lines and then I'd ship this and put the exception assignment handling in a new issue at lower priority.

Reply:
Ah yes, and a merge with master appears to be necessary